core/types/transaction.go

```diff
@@ -25,191 +26,174 @@ import (
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-//go:generate gencodec -type txdata -field-override txdataMarshaling -out gen_tx_json.go
```

Comment:
@lightclient you removed this gen-command, but you have also modified the `gen_tx_json.go` generated file. How did you generate that one? The command should be added to the definition-file, like here, so one can do e.g.  `go generate transaction.go ` to regenerate the files.

Reply:
Oh, I see. I manually updated the `gen_tx_json.go`.

Reply:
I see... This is a bit unconventional, I'll see if I can make it work differently, by generating files for legacy / acl separately, and have a custom marshal/unmarshal for `Transaction` which tests them both

Reply:
I don't know. It seems pretty tricky to auto-generate the corresponding code for this, so maybe it's indeed better to have it the way you've implemented it. If so, we should change the name from `gen_tx.json` to `transaction_marshalling.go` or something. 

@fjl what do you think?